src/ui/views/objectBrowser.ts

```diff
@@ -1090,7 +1090,15 @@ Do you want to replace it?`, item.name), { modal: true }, skipAllLabel, overwrit
           noLibList: true
         });
 
-        if (createResult.code !== 0) {
+
+        const isSuccess = createResult.code === 0;
+        if (isSuccess) {
+          const config = connection.getConfig();
+          const libraryList = [config.currentLibrary, ...config.libraryList].map(library => library.toUpperCase());
+          if (libraryList.includes(newLibrary)) {
```

Comment:
This does not work if the library created is not entered in uppercase...

```suggestion
          if (libraryList.includes(newLibrary.toUpperCase())) {
```